databases/week1/homework.sql

```diff
@@ -0,0 +1,53 @@
+-- Find out how many tasks are in the task table
+SELECT COUNT(*)
+FROM task;
+
+-- Find out how many tasks in the task table do not have a valid due date
+SELECT COUNT(*)
+FROM task
+WHERE due_date IS NULL;
+
+-- Find all the tasks that are marked as done
+SELECT * 
+FROM task
+JOIN status ON task.status_id = status.id
+WHERE status.name LIKE "%Done%";
+
+-- Find all the tasks that are not marked as done
+SELECT * 
+FROM task
+JOIN status ON task.status_id = status.id
+WHERE status.name NOT LIKE "%Done%";
```

Comment:
status.name != "Done";
_Pointer_
Use LIKE for pattern matches and perform an equality operation